build.py

```diff
@@ -1,12 +1,17 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-
+import os
+import platform
 from bincrafters import build_template_default
 
 if __name__ == "__main__":
 
-    builder = build_template_default.get_builder()
+    if platform.system() == "Linux":
+        command = os.environ["CONAN_DOCKER_ENTRY_SCRIPT"] + "; sudo apt-get -qq update && sudo apt-get -qq install -y python-minimal"
```

Comment:
Why is `python-minimal` needed ? It should already be installed as a conan requirement right ? Also, `python-minimal` seems to install python 2 on some distros, which end-of-life is passed

Reply:
Yes, the first problem I encountered is the missing `python2` dependency. https://github.com/bincrafters/conan-crashpad/runs/1746949242#step:5:654 on `testing/20200508`.

As I've learned, the chromium project's `luci/vpython` tool lets the scripts define their own virtual environment (much like pipenv or venv) to support scripts from the past. Looks like the tool `download_from_google_storage` is one that wants to run on py2 but the docker images that try to build the project don't include `python2` which is understandable. Other than installing manually in `build.py` I had no other idea to get it fixed.